pyspecProcScripts/phasing.py

```diff
@@ -167,8 +167,8 @@ def ph1_real_Abs(s, dw, ph1_sel=0, ph2_sel=1, fl=None):
         fl.plot(s_cost, ".")
     s_cost.smoosh(["vd", "phcorr"], "transients")
     ph1_opt = np.asarray(s_cost.argmin("transients").item())
-    logging.info(strm("THIS IS PH1_OPT", ph1_opt))
-    logging.info(strm("optimal phase correction", repr(ph1_opt)))
+    logging.debug(strm("THIS IS PH1_OPT", ph1_opt))
+    logging.debug(strm("optimal phase correction", repr(ph1_opt)))
```

Comment:
at some point, all these info messages become white noise, so I just made them all debug-level statements (so that they show up only in the log).  If there are 1 or 2 that you can justify appearing at the command line, feel free to change back, but there were just too many.

Once the message becomes so long, anyways, it's easiest to go look at the log file, which (unlike the command line) is searchable.

Reply:
agreed

Reply:
why are you making edits to functions that are not included in this pull request? doesn't that muddy the waters of the primary purpose of fixing/updating hermitian function test?